CanonicalLayouts/supporting-pane-compose/app/src/main/java/com/example/supportingpanecompose/ui/SupportingPaneSample.kt

```diff
@@ -47,53 +54,82 @@ private val data = mapOf(
     "flutter" to listOf("android", "desktop")
 )
 
+@ExperimentalMaterial3AdaptiveApi
 @Composable
-fun SupportingPaneSample(
-    windowSizeClass: WindowSizeClass,
-    displayFeatures: List<DisplayFeature>
-) {
+fun SupportingPaneSample() {
     var selectedTopic: String by rememberSaveable { mutableStateOf(data.keys.first()) }
+    val navigator = rememberSupportingPaneScaffoldNavigator()
 
-    SupportingPane(
-        main = {
-            Column(
-                horizontalAlignment = Alignment.CenterHorizontally,
-                modifier = Modifier.fillMaxSize()
+    SupportingPaneScaffold(
+        supportingPane = {
+            AnimatedPane(
+                modifier = Modifier
+                    .background(
+                        color = Color.LightGray.copy(alpha = .3f),
+                        shape = RoundedCornerShape(16.dp)
+                    )
+                    .padding(all = 16.dp)
             ) {
-                Text("Main Content", style = MaterialTheme.typography.titleLarge)
-                Text(selectedTopic)
-            }
-        },
-        supporting = {
-            Column(
-                modifier = Modifier.fillMaxSize()
-            ) {
-                Text("Related Content", style = MaterialTheme.typography.titleLarge)
+                Column {
+                    Text(
+                        "Related Content",
```

Comment:
I know it was already like this before, but I feel like we should always use string resources for all our samples instead of hardcoded strings. I'm fine if it's in another PR (might make sense to do a single PR to fix all of them). WDYT, @alexvanyo ?

Reply:
+1 agreed, doesn't need to block this PR but that'd be preferable